app/main.py

```diff
@@ -1,8 +1,52 @@
+from typing import Type
```

Comment:
The import of `Type` from `typing` is unnecessary here. You should remove it, as `Type` is not used correctly in the code.